forge-gui/res/cardsfolder/c/cavalcade_of_calamity.txt

```diff
@@ -2,6 +2,6 @@ Name:Cavalcade of Calamity
 ManaCost:1 R
 Types:Enchantment
 T:Mode$ Attacks | ValidCard$ Creature.powerLE1+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigDamage | TriggerDescription$ Whenever a creature you control with power 1 or less attacks, CARDNAME deals 1 damage to the player or planeswalker that creature is attacking.
-SVar:TrigDamage:DB$ DealDamage | Defined$ TriggeredDefender.Player & Valid Planeswalker.TriggeredDefender | NumDmg$ 1
+SVar:TrigDamage:DB$ DealDamage | Defined$ Player.TriggeredDefender & Valid Planeswalker.TriggeredDefender | NumDmg$ 1
```

Comment:
what is the reason there?

* `TriggeredDefender.Player` should be getDefined("TriggeredDefender") + Property Player
* `Player.TriggeredDefender` while this is all Player + TriggeredDefender check

imo the first one should be faster?
(i think we miss the `Player` property?)

Reply:
yea neither `Card.Card` nor `Player.Player` are valid syntax

feels stupid to change that imo

you'd prefer `TriggeredDefender.Opponent` instead?

Reply:
yeah, `TriggeredDefender.Opponent` is fine too